service.py

```diff
@@ -51,6 +51,16 @@ def showErrorNotification(message):
                                   xbmcgui.NOTIFICATION_ERROR, 5000)
 
 
+def get_user_input():
+    kb = xbmc.Keyboard('', 'Please enter the URL')
```

Comment:
I am not a native speaker but `a` like `Please enter a URL` would be better right?